### ISSUE TITLE
`items` schema of an `array` field is not displayed if `anyOf` is used

### ISSUE THREAD

**Issue by maciekpitucha** [PaloAltoNetworks/docusaurus-openapi-docs#343]
## Describe the bug

the following schema 

<details><summary>schema</summary>

```yaml
---
openapi: 3.0.3
servers:
  - url: https://
info:
  title: ""
  version: "1.0"
components:
  securitySchemes:
    basicAuth:
      type: http
      scheme: basic
paths:
  "/test":
    summary: summary
    post:
      summary: summary
      security:
        - basicAuth: []
      responses:
        "200":
          description: ""
          content:
            application/json:
              schema:
                type: object
                properties:
                  id:
                    type: integer
      requestBody:
        content:
          application/json:
            schema:
              type: object
              properties:
                items:
                  type: array
                  items:
                    anyOf:
                      - type: object
                        properties:
                          a:
                            type: integer
                      - type: object
                        properties:
                          b:
                            type: integer

```
</details>
is displayed as

![image](https://user-images.githubusercontent.com/114495507/201488687-005305ea-3472-439f-a88f-4a15d6e7dc40.png)

Schema of the `items` options is not displayed.

## Expected behavior

The schema is displayed in a way that elements in `anyOf` property of `items` are displayed. 

## Current behavior

Schema of the `items` options is not displayed.

## Steps to reproduce

1. Use the schema from this issue
1. `yarn docusaurus gen-api-docs all`
1. `yarn start`
1. go to the page that shows API schema

**Comment by sserrata** [comment]
Hi @maciekpitucha, thanks for reporting. This is a new edge case for our plugin that I believe we can support. My initial thought is that we can use `DiscriminatorTabs` to allow for choosing the object schema you want to reference inside the array.

**Comment by sserrata** [comment]
Hi @maciekpitucha, thanks again for bringing this issue to our attention. It actually pointed me to a bug that we hadn't noticed before which could have prevented schema properties from rendering in some cases.

Does the following match what you would expect to see?

<img width="400" alt="Screen Shot 2022-11-15 at 9 20 53 AM" src="https://user-images.githubusercontent.com/9343811/201942739-c1403f41-1617-4c0c-949d-e9b208a6735a.png">

**Comment by sserrata** [comment]
@maciekpitucha the fix is now available to test using canary release `0.0.0-515`.
